.circleci/config.yml

```diff
@@ -272,6 +272,45 @@ jobs:
           paths:
             - wp-calypso
 
+  translate:
+    <<: *defaults
+    steps:
+      - prepare
+      - run:
+          name: Extract strings
+          working_directory: wp-babel-makepot
+          command: npx https://github.com/Automattic/wp-babel-makepot "~/wp-calypso/{client,packages,apps}/**/*.{js,jsx,ts,tsx}" --ignore "**/node_modules/**,**/test/**,**/*.d.ts" --base "$HOME/wp-calypso" --output "$CIRCLE_ARTIFACTS/translate/calypso-strings.pot"
```

Comment:
Would it be possible to publish `wp-makepotjs` to the npm registry, add a devDependency on it, and invoke it here with `yarn run wp-babel-makepot`?

Reply:
The next step will most likely be moving it to the mono repo and publishing it to npm, but for now, the npx approach would allow revert back to previous state more easily, if anything goes wrong.